docs/faq.mdx

```diff
@@ -9,14 +9,32 @@ Below is a list of the frequently asked questions we have encountered from users
 
 ## General
 
-### I was using the older version of Superface, what should I do?
+### I use the older version of Superface, what should I do?
 
-If you were using the previous version of the Superface app, and using our pre-made Comlinks with OneSDK and the CLI, you can continue to do so for the time being. However, we do plan to deprecate the hosted Comlink functionality later this year so we recommend that you move to the latest version of OneSDK sooner rather than later. Find out more about our [suggested approach](./migrate.mdx).
+If you were using the previous version of the Superface app, and using our pre-made Comlinks with OneSDK and the CLI, you can continue to do so for the time being. However, we do plan to deprecate the hosted Comlink functionality later this year so we recommend that you move to the latest version of OneSDK and generate your own Comlink profiles from the documentation.
 
-### Is Superface ready for production use?
+### What is the license for Superface?
 
-Superface's OneSDK is currently in beta.
+Superface is licensed under the MIT license. You can check out the specific license details in the Readme for [OneSDK](https://github.com/superfaceai/one-sdk/blob/main/LICENSE), and the [Superface CLI](https://github.com/superfaceai/cli/blob/main/LICENSE).
+
+### Do I need to purchase a license for production use?
+
+No. There is no license required for production use of OneSDK at this time.
+
+### I have questions that aren't answered here, where else can I ask?
+
+You are welcome to reach out to us either through email on support@superface.ai, or head over to [GitHub Discussions](https://github.com/orgs/superfaceai/discussions) and ask your question there. The team are on hand to help you out.
+
+## OneSDK
+
+### Is Superface OneSDK ready for production use?
+
+Superface's OneSDK is currently in beta, so you should expect that changes will be made to future versions. We recommend that you test your application with any new version of OneSDK that you upgrade to before releasing to production.
 
 ## CLI
 
 Specific questions related to the Superface CLI.
+
+### How do I keep my CLI up to date?
+
+New releases and changelog information can be found in the [Announcements](https://github.com/orgs/superfaceai/discussions/categories/announcements) section of GitHub Discussions. You can also check for updates by running `brew update` and looking for updates to the `superface/cli` tap.
```

Comment:
The tap name is `superfaceai/cli` (was missing `...ai`).

This is because taps are just GitHub repos targeted as `[owner]/homebrew-[name]`, thus our [`superfaceai/homebrew-cli`](https://github.com/superfaceai/homebrew-cli) repo translates to `superfaceai/cli` tap

Reply:
Good catch. Updated.